### DISCUSSION TITLE
GitHub Actions Pricing

### DISCUSSION THREAD

**Discussion by mohammed-salam** [community/community#26033] [Actions]
<p>GitHub states the cost of data transfer with Actions is free from GitHub packages. Is this still free if Actions is being used on a self hosted machine? Also, if GitHub npm registry is used to proxy and retrieve a package from <a href="http://npmjs.com" rel="nofollow noopener">npmjs.com</a> within actions, is this still free? Thank you.</p>

**Comment by BrightRan** [comment, marked as answer]
<p>GitHub Actions usage is free for public repositories, and GitHub Packages usage is free for public packages. You can free to use minutes, storage and data transfer. The data transfer also is free no matter you are using self-hosted runners or GitHub-hosted runners.<br>
For private repositories and private packages, each GitHub account receives a certain amount of free minutes, storage and data transfer, depending on the product used with the account. If you want to increase your spending limits, you will be billed for any additional minutes, storage or data transfer, also called overages.</p>

**Comment by blowfishpro** [comment]
<p>Are the free limits per organization or per repository?</p>

**Comment by ltamrazov** [comment]
<p>You mentioned that the data transfer is free for self hosted runners on private repositories. What about the minutes? If I have a self-hosted runner on a private repository, will minutes bee free / unlimited?</p>
<p>Thank you</p>
